river/View.zig

```diff
@@ -379,3 +402,73 @@ pub fn unmap(self: *Self) void {
 
     root.startTransaction();
 }
+
+/// Change the opacity of a view by config.view_opacity_delta.
+/// If the target opacity was reached, return true.
+fn incrementOpacity(self: *Self) bool {
+    // TODO damage view when implementing damage based rendering
+    const config = self.output.root.server.config;
```

Comment:
Not sure if zig is smart enough to elide this copy yet.
```suggestion
    const config = &self.output.root.server.config;
```